settings/clientSettings.ts

```diff
@@ -4,7 +4,18 @@
 // bundled and shipped out to our users.
 
 import dotenv from "dotenv"
-dotenv.config()
+import fs from "fs"
+
+// only run the below code if we're in a node environment
+if (fs.statSync !== undefined) {
```

Comment:
it's still weird and surprisingly tricky to have some block of code that runs on node, but doesn't run in webpack. or, ideally, wouldn't even be bundled into the webpack bundle.